src/github-repo-event-notifier.coffee

```diff
@@ -32,12 +32,16 @@
 #   patcon
 #   parkr
 
+_ = require('lodash')
 inspect = (require('util')).inspect
 url = require('url')
 querystring = require('querystring')
-eventActions = require('./event-actions/all')
 eventTypesRaw = process.env['HUBOT_GITHUB_EVENT_NOTIFIER_TYPES']
 eventTypes = []
+customActions = {}
+try
+  customActions = require.main.require('../../../scripts/github/custom-actions')
```

Comment:
I'm concerned about this very narrow path choice – would it be possible to use a user-specified path instead?